tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml

```diff
@@ -0,0 +1,78 @@
+[setting.synthStataUsdcMaxMarketCollateral]
+defaultValue = "<%= parseEther(String(5_000_000)) %>"
+
+[setting.spotStrictStalenessTolerance]
+defaultValue = "60"
+
+[setting.sstatausdc_issuance_ratio]
+defaultValue = "<%= MaxUint256 %>"
+
+[setting.sstatausdc_liquidation_ratio]
+defaultValue = "<%= parseEther('1.1') %>"
+
+[setting.sstatausdc_liquidation_reward]
+defaultValue = "<%= parseEther('0.005') %>"
+
+[setting.sstatausdc_min_delegation]
+defaultValue = "<%= parseEther('100') %>"
+
+[invoke.SpotMarketProxy_createSynth_stataUSDC]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "owner"
+func = "createSynth"
+args = ["Synthetic Static aUSDC", "sStataUSDC", "<%= settings.owner %>"]
```

Comment:
original token symbol `stataBasUSDC` but I am not 100% sure if I want to rename everything added here to match that... As sepolia already deployed this will not be ideal to have non-matching deployments. 
I would definitely use exact symbol name everywhere when creating all these configs from scratch though.

Reply:
Yea, this was an oversight on my part, though counterpoint is if stataArbUSDC ended up on base somehow, we'd probably just use this same spot market to wrap it too.

Reply:
Thanks, no dramas